src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -69,43 +70,57 @@
 
         event.preventDefault();
 
+        if (isFormValid(btn)) {
+            window.setTimeout(() => btn.click(), 0);
```

Comment:
I would keep the comment with explanations why this has to be in timeout
Also could you change `window` to `global`.

Reply:
done